src/main/webapp/transcript.js

```diff
@@ -67,9 +67,9 @@ function appendTextToList(transcriptLine, ulElement) {
   const startDate = new Date(transcriptLine.start);
   const endDate = new Date(transcriptLine.end);
   const startTimestamp = `${startDate.getHours()}:${startDate.getMinutes()}:${
-      startDate.getSeconds()}`;
+    startDate.getSeconds()}`;
   const endTimestamp =
-      `${endDate.getHours()}:${endDate.getMinutes()}:${endDate.getSeconds()}`;
+    `${endDate.getHours()}:${endDate.getMinutes()}:${endDate.getSeconds()}`;
```

Comment:
Maybe these could be a util function? dateToTimestampString

Reply:
I'm creating a Util class in a different pull request right now. I added an issue for this #94 so I won't forget. Thanks!

Reply:
Sounds good!